xen/xenops_interface.ml

```diff
@@ -433,6 +433,8 @@ module Vif = struct
 
 	type id = string * string
 
+	type static_ip_setting = (string * string) list
```

Comment:
type ip_configuration =
    | Unspecified
    | Static of string * string list * string option (* enabled: 1 or 0, a list of CIDRs and optionally a gateway *)

I add a field, type is string, to Vif.Static.

Then in somewhere, I do 
"ip_configuration = Vif.Static ("1", ["192.168.1.100/24"; "192.168.1.200/24"], (Some "192.168.1.1"));"

The compiler reports 

"Error: The constructor Vif.Static expects 2 argument(s),
       but is applied here to 3 argument(s)
Command exited with code 2."

Does Vif.Static only support 1 or 2 fields?